app/assets/stylesheets/help.css.scss

```diff
@@ -91,6 +91,18 @@ ul#help_nav {
       border-radius: 0px 0px 4px 4px;
       background-color: white;
       padding: 10px 20px;
+
+      div.help-chat-icons{
+        text-align: center;
+        font-size: 50px;
+        line-height: 70px;
+
+        i.entypo{
```

Comment:
Nesting should be no greater than 4, but was 5<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 5<br>Opening curly brace `{` should be preceded by one space